docker-compose-wsl.yml

```diff
@@ -0,0 +1,75 @@
+version: "3"
+services:
+  prisoner-offender-search:
+    image: quay.io/hmpps/prisoner-offender-search:latest
+    networks:
+      - hmpps
+    container_name: prisoner-offender-search
+    ports:
+      - "8080:8080"
+    healthcheck:
+      test: ["CMD", "curl", "-f", "http://localhost:8080/health"]
+    environment:
+      - SERVER_PORT=8080
+      - SPRING_PROFILES_ACTIVE=dev
+      - API_BASE_URL_OAUTH=http://oauth-server:8080/auth
+      - API_BASE_URL_NOMIS=http://prison-api:8080
+      - SQS_ENDPOINT_URL=http://localstack:4566
+      - ELASTICSEARCH_HOST=localstack
+      - ELASTICSEARCH_PROXY_URL=http://localstack:4571
+      - OAUTH_CLIENT_ID=prisoner-offender-search-client
+      - OAUTH_CLIENT_SECRET=clientsecret
+
+  prisonapi:
+    image: quay.io/hmpps/prison-api:latest
+    networks:
+      - hmpps
+    container_name: prisonapi
+    depends_on:
+      - oauth
+    ports:
+      - "8093:8080"
+    healthcheck:
+      test: ["CMD", "curl", "-f", "http://localhost:8080/health"]
+    environment:
+      - SERVER_PORT=8080
+      - SPRING_PROFILES_ACTIVE=nomis-hsqldb
+
+  oauth:
+    image: quay.io/hmpps/hmpps-auth:latest
+    networks:
+      - hmpps
+    container_name: oauth
+    ports:
+      - "8090:8080"
+    healthcheck:
+      test: ["CMD", "curl", "-f", "http://localhost:8080/auth/health"]
+    environment:
+      - SERVER_PORT=8080
+      - SPRING_PROFILES_ACTIVE=dev
+
+  localstack:
+    image: localstack/localstack:latest
```

Comment:
we normally specify explicit dependencies for third parties to prevent latest versions from breaking without us realising.
So
```
    image: localstack/localstack:0.12.10
``` 
would be better.

Reply:
[_See latest commit_](https://github.com/ministryofjustice/prisoner-offender-search/pull/166/commits/1f81f57f58238a61d9ef1e17e28b6cb8afcc0f33).